src/pages/workspace/reimburse/WorkspaceReimburseView.tsx

```diff
@@ -142,42 +113,47 @@ function WorkspaceReimburseView(props) {
                 <View style={[styles.mv3, styles.flexRow, styles.flexWrap]}>
                     <Text>{translate('workspace.reimburse.trackDistanceCopy')}</Text>
                     <OfflineWithFeedback
-                        pendingAction={lodashGet(distanceCustomUnit, 'pendingAction') || lodashGet(distanceCustomRate, 'pendingAction')}
+                        pendingAction={distanceCustomUnit?.pendingAction ?? distanceCustomRate?.pendingAction}
                         shouldShowErrorMessages={false}
                         style={styles.w100}
                     >
                         <MenuItemWithTopDescription
                             title={currentRatePerUnit}
                             description={translate('workspace.reimburse.trackDistanceRate')}
                             shouldShowRightIcon
-                            onPress={() => Navigation.navigate(ROUTES.WORKSPACE_RATE_AND_UNIT.getRoute(props.policy.id))}
+                            onPress={() => Navigation.navigate(ROUTES.WORKSPACE_RATE_AND_UNIT.getRoute(policy?.id ?? ''))}
                             wrapperStyle={[styles.sectionMenuItemTopDescription, styles.mt3]}
-                            brickRoadIndicator={(lodashGet(distanceCustomUnit, 'errors') || lodashGet(distanceCustomRate, 'errors')) && CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR}
+                            brickRoadIndicator={(distanceCustomUnit?.errors ?? distanceCustomRate?.errors) && CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR}
                         />
                     </OfflineWithFeedback>
                 </View>
+                <OfflineWithFeedback
+                    pendingAction={distanceCustomUnit?.pendingAction ?? distanceCustomRate?.pendingAction}
+                    shouldShowErrorMessages={false}
+                >
+                    <MenuItemWithTopDescription
+                        title={currentRatePerUnit}
+                        description={translate('workspace.reimburse.trackDistanceRate')}
+                        shouldShowRightIcon
+                        onPress={() => Navigation.navigate(ROUTES.WORKSPACE_RATE_AND_UNIT.getRoute(policy?.id ?? ''))}
+                        wrapperStyle={[styles.mhn5, styles.wAuto]}
+                        brickRoadIndicator={(distanceCustomUnit?.errors ?? distanceCustomRate?.errors) && CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR}
+                    />
+                </OfflineWithFeedback>
```

Comment:
Why was it added, the code looks really similar to the code block above, maybe you did a mistake during conflicts resolution?

Reply:
You are right! It was badly conflicts merge! Thank you!